src/Classes/Entities/StageEntity.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Portal\Entities;
+
+class StageEntity extends ValidationEntity implements \JsonSerializable
+{
+    protected $id;
+    protected $title;
+    protected $order;
+
+    public function __construct(string $id = null, string $title = null, string $order = null)
+    {
+        $this->id = ($this->id ?? $id);
+        $this->title = ($this->title ?? $title);
+        $this->order = ($this->order ?? $order);
+        $this->sanitiseData();
+    }
+
+    /**
+     * Returns private properties from object.
+     *
+     * @return array|mixed
+     */
+    public function jsonSerialize()
+    {
+
+        return [
+            'id' => $this->id,
+            'title' => $this->title,
+            'order' => $this->order
+        ];
+    }
+
+    /**
+     * Will sanitise all the fields for a stage.
+     */
+    private function sanitiseData()
+    {
+        $this->id = (int) $this->id;
+        $this->title = self::sanitiseString($this->title);
+        $this->order = (int) $this->order;
+    }
```

Comment:
does this work? Does it need to return anything?

Reply:
yes, it is replacing the contents of the parameters with santized versions of themselves.